src/main/java/at/laborg/briss/BrissGUI.java

```diff
@@ -78,10 +40,53 @@
 import java.io.File;
 import java.io.IOException;
 import java.net.URI;
-import java.text.ParseException;
 import java.util.ArrayList;
 import java.util.List;
 
+import javax.swing.BoxLayout;
+import javax.swing.ImageIcon;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JMenu;
+import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JPasswordField;
+import javax.swing.JProgressBar;
+import javax.swing.JScrollPane;
+import javax.swing.KeyStroke;
+import javax.swing.ScrollPaneConstants;
+import javax.swing.SwingUtilities;
+import javax.swing.SwingWorker;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+import org.jpedal.exception.PdfException;
+
+import com.itextpdf.text.DocumentException;
+
+import at.laborg.briss.exception.CropException;
+import at.laborg.briss.gui.HelpDialog;
+import at.laborg.briss.gui.MergedPanel;
+import at.laborg.briss.gui.WrapLayout;
+import at.laborg.briss.model.ClusterDefinition;
+import at.laborg.briss.model.CropDefinition;
+import at.laborg.briss.model.PageCluster;
+import at.laborg.briss.model.PageExcludes;
+import at.laborg.briss.model.WorkingSet;
+import at.laborg.briss.utils.BrissFileHandling;
+import at.laborg.briss.utils.ClusterCreator;
+import at.laborg.briss.utils.ClusterRenderWorker;
+import at.laborg.briss.utils.DesktopHelper;
+import at.laborg.briss.utils.DocumentCropper;
+import at.laborg.briss.utils.FileDrop;
+import at.laborg.briss.utils.PDFReaderUtil;
+import javafx.application.Platform;
+import javafx.embed.swing.JFXPanel;
+import javafx.stage.FileChooser;
```

Comment:
Seems like we have different settings for the imports. We should align there somehow, otherwise this will change back and forth between commits.

Reply:
Agreed. I just use CMD + SHIFT + O to organize imports automatically in Eclipse. We can use checkstyle to uniformize the import order (and forbid underscores in private methods 😉 ). We can progressively add some rules to it in the future. What do you think?